### PR TITLE
Add null platform for emscripten

### PR DESCRIPTION
**Pull request by cactorium**
To help with https://github.com/tomaka/glutin/pull/857 Sorry for being so bothersome!

### CODE REVIEW

**Comment by tomaka** [comment]
Thanks!

It shouldn't be `emscripten` through. Instead you should name the module `null`, and put that big `#[cfg]` block over `mod null;` in `src/platform/mod.rs`.

File: src/platform/emscripten/mod.rs [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+pub struct Window;
+
+impl Window {
+    #[inline]
+    pub fn new(_: &WindowAttributes,
+               _: &PlatformSpecificWindowBuilderAttributes)
+                -> Result<Window, CreationError> {
+        // getting the default values of attributes
+        Ok(Window)
```
Comment (tomaka) [inline src/platform/emscripten/mod.rs]: Please return `Err(CreationError::NotSupported)` instead.

**Comment by cactorium** [comment]
Fixed!

**Comment by tomaka** [comment]
Thanks! Will merge when travis passes.

**Comment by tomaka** [comment]
Wait, I didn't notice but you made the pull request against master.
Could you do it against the `dev-0.5` branch of winit?
If you're not familiar enough with git to do that I can also do it myself.

**Comment by cactorium** [comment]
Oh my bad, I guess it got mixed up somehow

**Pull request closed without merging**